testutil/singlevm/setup.sh

```diff
@@ -146,6 +147,8 @@ openssl req -x509 -nodes -days 365 -newkey rsa:2048 -keyout controller_key.pem -
 #Copy the certs
 sudo cp -f controller_key.pem /etc/pki/ciao
 sudo cp -f controller_cert.pem /etc/pki/ciao
+sudo ln -s /etc/pki/ciao/controller_key.pem /etc/pki/ciao/ciao-image-key.pem
+sudo ln -s /etc/pki/ciao/controller_cert.pem /etc/pki/ciao/ciao-image-cacert.pem
```

Comment:
Since cert management is complicated, and good security depends on it, I like the idea of the single VM modelling best practices and "real" workflows.  Otherwise we'd not call ciao-cert or openssl at all for efficiency and just checkout a pre-canned set of test-keys like some other OS's and build systems do.